# Hypothesis Testing Exercise - BuyersRatio data

Problem Statement :
Sales of products in four different regions is tabulated for males and females. Find if male-female buyer rations are similar across regions

# Business Problem

Is the male-female buyer rations are similar across regions

# Data Description

α == 0.05 (95% Confidence)

Y == Discrete X == Discrete

Since there are more than 2 variable we will perform Chi-Square test

In [2]:
import pandas as pd
import numpy as np
from scipy import stats

In [3]:
ls

 Volume in drive C has no label.
 Volume Serial Number is C2AC-6915

 Directory of C:\Users\user\EXCELR\Assignments\Hypothesis testing

04/01/2022  08:50 PM    <DIR>          .
03/30/2022  10:44 AM    <DIR>          ..
04/01/2022  08:46 PM    <DIR>          .ipynb_checkpoints
03/30/2022  10:43 AM                87 BuyerRatio.csv
03/30/2022  10:43 AM            13,423 Costomer+OrderForm.csv
03/30/2022  10:43 AM               525 Cutlets.csv
04/01/2022  08:41 PM             8,113 Hypothesis Testing Exercise - Costomer+OrderForm data.ipynb
03/30/2022  11:10 AM             6,898 Hypothesis Testing Exercise - Cutlets.mtw data.ipynb
03/30/2022  11:26 AM            10,275 Hypothesis Testing Exercise - LabTat data.ipynb
03/30/2022  10:43 AM            61,688 Hypothesis_Testing_Assignment.pptx
03/30/2022  10:43 AM             3,487 LabTAT.csv
04/01/2022  08:50 PM             1,686 Untitled.ipynb
               9 File(s)        106,182 bytes
               3 Dir(s)  171,783,688,192 bytes free


# Chi-Square Test

In [4]:
df=pd.read_csv("BuyerRatio.csv")

In [5]:
df.head()

Observed Values  East  West  North  South
0           Males    50   142    131     70
1         Females   435  1523   1356    750

H0 == The male-female buyer rations are similar across regions
H1 == The male-female buyer rations are not similar across regions

In [10]:
stats.chi2_contingency([df['East'], df['West'], df['North'], df['South']])

(1.5959455386610577,
 0.6603094907091882,
 3,
 array([[  42.76531299,  442.23468701],
        [ 146.81287862, 1518.18712138],
        [ 131.11756787, 1355.88243213],
        [  72.30424052,  747.69575948]]))

P value of Chi-Square test == 0.66 > α

Thus HO is accepted.

The male-female buyer rations are similar across regions